MAUI/ListView/item-drag-and-drop.md

```diff
@@ -0,0 +1,485 @@
+</Grid>            
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+public partial class MainPage : ContentPage
+{
+  public MainPage()
+  {
+    InitializeComponent();
+    var maingrid = new Grid();
+    maingrid.RowDefinitions.Add(new RowDefinition { Height = new GridLength(40) });
+    maingrid.RowDefinitions.Add(new RowDefinition { Height = new GridLength(1, GridUnitType.Star) });
+    var grid = new Grid();
+    var headerLabel = new Label()
+    {
+        Text = "To Do Items"
+    };
+    var stackLayout = new StackLayout();
+    stackLayout.SetBinding(StackLayout.IsVisibleProperty, new Binding("IsVisible"));
+    var image = new Image() { Source = "delete.png" };
+    var deleteLabel = new Label() { Text = "DeleteItem" };
+    stackLayout.Children.Add(image);
+    stackLayout.Children.Add(deleteLabel);
+    grid.Children.Add(headerLabel);
+    grid.Children.Add(stackLayout);
+    var listView = new SfListView()
+    {
+        DragStartMode = DragStartMode.OnHold,
+        ItemSize = 60,
+        SelectionMode = Syncfusion.Maui.ListView.SelectionMode.None,
+        BackgroundColor = Color.FromHex("#FFE8E8EC")
+    };
+    listView.ItemsSource = viewModel.ToDoList;
+    maingrid.Children.Add(grid);
+    maingrid.SetRow(grid, 0);
+    maingrid.Children.Add(listView);
+    maingrid.SetRow(listView, 1);
+    this.Content = maingrid;
+  }
+}
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# %}
+private async void ListView_ItemDragging(object sender, ItemDraggingEventArgs e)
+{
+  var viewModel = this.listView.BindingContext as ViewModel;
+
+  if (e.Action == DragAction.Start)
+  {
+    viewModel.IsVisible = true;
+    this.headerLabel.IsVisible = false;
+  }
+
+  if(e.Action == DragAction.Dragging)
+  {
+    var position = new Point(e.Position.X - this.ListView.Bounds.X, Math.Abs(e.Position.Y - this.ListView.Bounds.Y));
+    if ((this.Stack.Bounds.Y < position.Y) && (this.Stack.Bounds.Y + this.Stack.Height) > position.Y)
+    {
+      viewModel.IsVisible = true;
+      this.deleteLabel.TextColor = Colors.Red;
+    }
+    else
+      this.deleteLabel.TextColor = Colors.White;
+  }
+
+  if(e.Action == DragAction.Drop)
+  {
+    var position = new Point(e.Position.X - this.ListView.Bounds.X, Math.Abs(e.Position.Y - this.ListView.Bounds.Y));
+    if ((this.Stack.Bounds.Y < position.Y) && (this.Stack.Bounds.Y + this.Stack.Height) > position.Y)
+    {
+      await Task.Delay(100);
+      viewModel.ToDoList.Remove(e.DataItem as ToDoItem);
+    }
+    viewModel.IsVisible = false;
+    this.deleteLabel.TextColor = Colors.White;
+    this.headerLabel.IsVisible = true;
+  }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Delete item while drop in listview](Images/item-drag-and-drop/maui-listview-draganddel.gif)
+
+## Skip dragging item into another group
+
+To skip dragging from one group to another group, handle the `ItemDragging` event based on the conditions of `Action` and `Bounds` event arguments.
```

Comment:
To skip dragging from one group to another group, handle the `ItemDragging` event based on the conditions of the `Action` and `Bounds` event arguments.

Reply:
Addressed the concerns provided.